lib/src/test/java/uk/gov/di/ipv/cri/kbv/api/util/KBVAnswerStorageMapperTest.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.di.ipv.cri.kbv.api.util;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.di.ipv.cri.kbv.api.domain.KBVAnswerItem;
+import uk.gov.di.ipv.cri.kbv.api.domain.KbvQuestion;
+import uk.gov.di.ipv.cri.kbv.api.domain.KbvQuestionOptions;
+import uk.gov.di.ipv.cri.kbv.api.domain.QuestionsResponse;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class KBVAnswerStorageMapperTest {
+    @Test
+    void shouldMapQuestionResponseQuestionOptionsToKBVAnswerItems() {
```

Comment:
If it's not going to take too long then it's probably worth adding another unit test to cover the scenario where a question has no options